### DISCUSSION TITLE
modification of UI of Backend NocoDB

### DISCUSSION THREAD

**Discussion by JS-PradnyaC** [nocodb/nocodb#3024] [General]
i have setup nocodb as per development setup steps
in order to make changes to frontend GUI there is package called nc-gui
i would like to know if we want to do any modification to backend UI, how to do the same?

**Comment by o1lab** [comment]
It will be nocodb package

**Reply by JS-PradnyaC** [reply]
in nocodb package, where is the UI files related folder? for eg i want to change text on sign up form. where do i get this file

**Reply by JS-PradnyaC** [reply]
i found it under v1-legacy folder. thanks :)
